src/database/test/DatabaseTests.cpp

```diff
@@ -353,14 +353,25 @@ TEST_CASE("schema test", "[db]")
 class SchemaUpgradeTestApplication : public TestApplication
 {
   public:
-    SchemaUpgradeTestApplication(VirtualClock& clock, Config const& cfg)
+    using PreUpgradeFunc = std::function<void(SchemaUpgradeTestApplication&)>;
+
+  private:
+    PreUpgradeFunc const preUpgradeFunc;
```

Comment:
Nitpick: code-style conventions, should be `mPreUpgradeFunc`

Reply:
Fixed.